app/assets/javascripts/partners.js.coffee

```diff
@@ -0,0 +1,5 @@
+jQuery ->
+  $("#partner_start_date").datepicker({
+    format: "yyyy-mm-dd"
+    autoclose: true
+  })
```

Comment:
Is there a way to generalize this so that we don't have to add JS files for other models' views with date fields?